src/views/partials/main/title.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="container mb-2">
   <div class="row">
     <div class="col-12">
-      <p class="h1 font-italic my-4">{{title}}</p>
+      <p class="h1 my-4">{{title}}</p>
       <p class="lead">{{subtitle}}</p>
       <hr>
```

Comment:
Maybe remove this `<hr>` and add `border-bottom`?
https://getbootstrap.com/docs/4.3/utilities/borders/#additive

Reply:
Fixed it